test/app-tests/loading.js

```diff
@@ -343,6 +346,40 @@ describe('loading:', function () {
             }).done(done, done);
         });
 
+        it("registers correctly with a Riot Team Server", function() {
+            sdk.setFetch(httpBackend.fetchFn); // XXX: ought to restore this!
```

Comment:
no... it goes through [here](https://github.com/matrix-org/matrix-react-sdk/pull/1098/files#diff-1fdf421c05c1140f6d71444ea2b27638R32).

(it turns out that if I `import RtsClient` directly, then I get a *different* RtsClient to that being used by the react-sdk, which means that replacing its fetchFn doesn't work. I think that's due to https://github.com/webpack/webpack/issues/2740. In the end I decided it probably makes sense to replace the fetchfn for the whole of the sdk anyway.)